src/Controllers/UserController.cs

```diff
@@ -66,10 +66,19 @@ public async Task<ActionResult> GetACSUser()
         [ProducesResponseType(typeof(IdentityMapping), StatusCodes.Status201Created)]
         public async Task<ActionResult> CreateACSUser()
         {
-            string acsUserId = await _acsService.CreateACSUserIdentity();
```

Comment:
could you take care of the warnings above?

Reply:
Run `dotnet format` or press `ctrl + .` to call quick action